dev/SapphireTest.php

```
@@ -670,29 +670,36 @@ public function assertDOSContains($matches, $dataObjectSet) {
 	 * @param mixed $dataObjectSet The {@link SS_List} to test.
```

This is a bugfix to assertDOSEquals silently passing if $dataObjectSet is null.